app/src/userpages/components/ProductsPage/Members/index.jsx

```diff
@@ -134,10 +136,10 @@ const Members = () => {
         setApproving(false)
         selection.none()
 
-        if (loadMembersRef.current) {
+        if (isMounted() && loadMembersRef.current) {
```

Comment:
I guess we should put `isMounted()` before the above `setApproving` and `selection.none()`.

Reply:
Ah, true 👍  Fixed